### ISSUE TITLE
Build fails with "Can't find stylesheet to import." if injectGlobalPaths is used

### ISSUE THREAD

**Issue by kerosin** [ionic-team/stencil-sass#49]
## Current Behavior
If using injectGlobalPaths parameter
```
plugins: [
    sass({
      injectGlobalPaths: ['src/globals/variables.scss'],
    }),
  ],
```
the build fails with
```
[ ERROR ]  sass error: ...tencil-example/libs/web-components/src/components/my-component/my-component.scss:1:9
           Can't find stylesheet to import.
```
If I leave out the parameter, the builds succeeds.I suspect the path resolution not working correctly on my nx workspace, because it works on a simple stencil project.

## Expected Behavior
using injectGlobalPaths parameter should work without causing issues

## Context (Environment)
- nx 11 workspace
- @stencil/core 2.3.0
- @stencil/sass: 1.4.1
- Windows

## Steps to Reproduce
1. clone https://github.com/kerosin/nx-stencil-example
2. call `nx build web-components`

**Comment by kerosin** [comment]
reason is that ```context.config.rootDir``` points to `nx-stencil-example\dist\libs\web-components` instead of `nx-stencil-example/libs/web-components/stencil.config.ts`

So [resolving the paths injectGlobalPaths](https://github.com/ionic-team/stencil-sass/blob/b1cc372e98837f63d78b36226af072d083659fbc/src/util.ts#L44-L54) is errornous and causes an error on the dart sass package.

**Comment by janrembold** [comment]
@jase88  Is there any workaround for this issue? For me this feature is completely broken right now :(

**Comment by corysmc** [comment]
I'm also having issues with this.
nx workspace using stencil, and I'm getting the same error when trying to inject global paths. Anyone come up with a workaround?

**Comment by jase88** [comment]
So what this global path does is just adding your globals to the head of each scss file with `@use myGlobal`. 

So the only workaround I can think of would be a own script that automatically writes your global files at the top :/

**Comment by corysmc** [comment]
Thanks @jase88 that helps!

**Comment by tfrijsewijk** [comment]
You could try version 2.0.0-0, which is released 13 days ago but with dist-tag `next` instead of `latest`

**Comment by dgonzalezr** [comment]
I just faced the same issue while trying to move my NPM monorepo to Nx workspaces, what worked for me with `@stencil/sass` v1.5.2 was concatenating the path with `__dirname`:

```js
plugins: [
  sass({
    injectGlobalPaths: [`${__dirname}/src/globals/variables.scss`],
  }),
],
```

Hope it helps 🙂

**Comment by Wvduijn** [comment]
@dgonzalezr This works for me! Nice man thanks!

**Comment by rallz-dev** [comment]
Even with the solution proposed by @dgonzalezr , I'm getting the same "Can't find stylesheet to import" error. Is there any updates on this issue?

**Comment by jaredtbates** [comment]
We are seeing the same issue, but fixed on Mac/Linux when using the fix from @dgonzalezr. It still doesn't work on Windows though.

**Comment by dgonzalezr** [comment]
> This fixed the issue on Windows:
> 
> ```
> import path from 'path';
> 
> ...
> 
>   plugins: [
>     sass({
>       injectGlobalPaths: [
>         path
>           .resolve(__dirname, `src/styles/variables.scss`)
>           .replace(/\\/g, '/'),
>       ],
>     }),
>   ],
> ```

Indeed, my solution only works on Unix, I was struggling with the same issues on Windows, @jaredtbates you've "saved my life" 😆 )))

**Comment by damorossi** [comment]
Hello, guys, if you have worked before with sass, the imports are usually @import 'my-sass-file', without extension. Also many times we are not used to use './......' in our imports, for which this could be helpful:
for example, my file is called colors.scss
 plugins: [
    sass({
      injectGlobalPaths: [
        './src/global/colors'
      ],
    }),
  ],
  
  
  I hope it helps.

**Comment by NoelLH** [comment]
Has anybody else seen this crop up just with quite new `@stencil/sass`es?

I had some paths without prefixes, and with file extensions, working in `injectGlobalPaths` for some time. It looks like when I pin to `@stencil/sass@3.0.12` it works again. New versions including 3.2.0 exhibit errors like above – so I'm not sure whether it is better to continue here or open a new issue?

I tried both workaround ideas and they don't seem to change anything.
